.github/workflows/ci.yml

```diff
@@ -25,6 +25,11 @@ jobs:
         run: |
           docker build -t buildenv - < docker-build-env/Dockerfile
 
+      - name: "Check code format"
+        run: |
+          docker run --rm -u "$UID" -v "$PWD:/src:rw" -w /src buildenv bash -c \
```

Comment:
Just checked the output of the jobs.

Seems like this is ran for every combination of compiler and build_type.
I guess running it once should be enough.

Reply:
Done.